### PR TITLE
Few UI bug fix

### LINKED ISSUE #537: Bug / line numbers collapsed with the code
image

### LINKED ISSUE #556: Bug the add project button get collapsed due long workspace name
image

### PR DESCRIPTION
**Pull request by robonetphy**
image
resolves #537 
resolves #556 
Resolves #537

### CODE REVIEW

**Review by GeekaN2 — APPROVED** [review]

**Review by talyguryn — APPROVED** [review]

File: src/components/aside/WorkspaceInfo.vue [hunk 1]
```diff
@@ -119,7 +116,7 @@ export default Vue.extend({
         },
         popoverProps:{
           top: '65px',
-          left: '208px',
+          left: '215px',
```
Comment (talyguryn) [inline src/components/aside/WorkspaceInfo.vue:119]: still not centered 

image

Comment (talyguryn) [reply src/components/aside/WorkspaceInfo.vue:119]: is there any way to detect position automatically?

**Review by talyguryn — CHANGES_REQUESTED** [review]

File: src/components/aside/WorkspaceInfo.vue (hunk 1 above)
Comment (robonetphy) [reply src/components/aside/WorkspaceInfo.vue:119]: @talyguryn I tried to use the event method `getboundingclientrect()` to get the coordinates of the `div` container but the coordinates which I am getting is not accurate.
I also tried Vue `refs` but the problem remains the same.

**Review by neSpecc — APPROVED** [review]

**Review by talyguryn — APPROVED** [review]

File: src/components/aside/WorkspaceInfo.vue (hunk 1 above)
Comment (neSpecc) [reply src/components/aside/WorkspaceInfo.vue]: I've improved that logic. now it will be computed automatically

**Pull request merged**